src/euphorie/client/browser/templates/panel-contents-preview.pt

```diff
@@ -57,11 +57,13 @@
                       <ul class="risks">
                         <li class="risk"
                             tal:define="
+                              description python:view.get_description(node);
                               legal_references python:view.get_legal_references(node);
                               solutions python:view.get_solutions(node);
                             "
                         >
-                            ${node/title}
+                          <em>${node/title}</em>
+                          <p tal:replace="structure description"></p>
```

Comment:
Displaying content that can be entered by a user (either end user or tool creator) directly as rendered HTML is _verboten_ because of potential XSS attacks. See e.g. e9746b7023d55bb126a8aa484dc56fb3a440beff
```suggestion
                          <p tal:replace="structure python:webhelpers.get_safe_html(description)"></p>
```

Please also apply the same `get_safe_html`  check to the other places where `structure` is used (for **solution** and **legal_references** below).